README.md

```diff
@@ -14,7 +14,8 @@ As such, Optimum enables users to efficiently use any of these platforms with th
 🤗 Optimum aims at providing more diversity towards the kind of hardware users can target to train and finetune their models.
 
 To achieve this, we are collaborating with the following hardware manufacturers in order to provide the best transformers integration:
-- [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. [More information here](https://www.graphcore.ai/products/ipu)
+- [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. [More information here.](https://www.graphcore.ai/products/ipu)
+- [Habana Gaudi Processor (HPU)](https://github.com/huggingface/optimum-habana) - HPUs are designed to maximize training throughput and efficiency. [More information here.](https://habana.ai/training/)
```

Comment:
Just one minor suggestion. I see the introduction of IPUs focuses on what it is and the one of HPUs focuses more on what it does, is that intended?

Reply:
Your observation is right on the money but that was not intended XD
I thought this sentence from Habana docs sums up well the purpose of HPUs so I used it here.